style/styles.css

```diff
@@ -453,6 +451,128 @@ header {
   margin-top: 6px;
 }
 
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 50px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.6);
+}
+
+.modal-content {
+  background-color: #fefefe;
```

Comment:
- [ ] Good job so far 👏 Your popup can be opened and closed. However, you still need to work on the alignment and styling of its content because they are not looking very well. Try to see the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/180576410-f31bde1d-fa9b-4078-b476-547eaf8fd278.png)

**Figma**
![image](https://user-images.githubusercontent.com/90222110/180576585-db588f71-10c4-4336-b968-f3a2651ddac5.png)

Please rework on the size and styling of different contents in your popup to make sure that they are visible in the browser current window without user needing to scroll down in the popup. 

**NOTE:** There should be no vertical scrollbar on the popup window. Make sure that all the content are visible in the current window 😃 
💡 **HINT** Try to reduce the size of the image and also of the paragraph text. Reduce the spacing between contents but keep referring to the design given